SwiftUIBasics/Views/PlansView.swift

```diff
@@ -1,18 +1,100 @@
 //
-//  PlansView.swift
-//  SwiftUIBasics
+//  ContentView.swift
+//  ExercisesUI
 //
-//  Created by Diplomado on 09/12/23.
+//  Created by Diplomado  on 09/12/23.
 //
 
 import SwiftUI
 
-struct PlansView: View {
+struct ContentView2: View {
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack(alignment:.center){
+            Text("Choose")
+                .font(.system(size: 32, weight: .black, design: .serif))
+                .fontDesign(.rounded)
+            Text("Your Plan")
+                .font(.system(size: 32, weight: .black, design: .serif))
+                .fontDesign(.rounded)
+            HStack {
+                CardView(iconName: "", title: "Basic", description: "Per month", price: "$9")
```

Comment:
your card view could have an icon name with a default value and that could be skipped when the card is instantiated